src/stats/bayesian/model.rs

```diff
@@ -106,6 +88,30 @@ where
         }
     }
 
+    pub fn likelihood(&self) -> &L {
+        &self.likelihood
+    }
+
+    pub fn likelihood_mut(&mut self) -> &L {
```

Comment:
Shouldn't this return `&mut L` akin to the returns of the two other mutable getters?

Reply:
Right, my bad